.github/ISSUE_TEMPLATE/new-feature.yml

```
@@ -0,0 +1,26 @@
+name: New Content
+description: Add documentation for a new feature or apply content suggestions
+labels: ['new-content']
+body:
+  - type: textarea
+    id: issue-description
+    attributes:
+      label: Description
+      description: Outline what needs to be changed.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: issue-context
+    attributes:
+      label: Context
+      description: Provide any applicable background information.
+      placeholder:
+    validations:
+      required: true
```

```suggestion
      description: Provide a detailed description of the content suggestion or new feature documentation, including any context.
      placeholder:
    validations:
      required: true
```

Consolidate, similar to the "Fix content" template.